docs/tutorial/kubernetes.md

```diff
@@ -267,7 +267,7 @@ is doing:
 ## Next steps
 
 - In addition to `run_function`, Meadowrun provides [other
-  entrypoints](../entry_points). The most important of these is
+  entry points](../entry_points). The most important of these is
```

Comment:
I think we should update the links to the pages you moved in all of the Next steps sections? Things will still work because of the redirects, but seems better to link to the actual page. I also added an mkdocs-linkcheck packages so you can now run `poetry run mkdocs-linkcheck --sync -r docs`